include/linux/regmap.h

```diff
@@ -641,6 +645,10 @@ struct regmap *__devm_regmap_init_sdw(struct sdw_slave *sdw,
 				 const struct regmap_config *config,
 				 struct lock_class_key *lock_key,
 				 const char *lock_name);
+struct regmap *__devm_regmap_init_sdw_mbq(struct sdw_slave *sdw,
+					  const struct regmap_config *config,
+					  struct lock_class_key *lock_key,
+					  const char *lock_name);
```

Comment:
where are these functions going to be called from? Will that code have to compile with `CONFIG_REGMAP_SOUNDWIRE_MBQ` unselected? Then we might need dummies for `#ifndef CONFIG_REGMAP_SOUNDWIRE_MBQ`?

Reply:
there is a wrapper used by codec drivers. The codec drivers use 'select REGMAP_SOUNDWIRE_MBQ' so there's no need to a fallback.

Reply:
no change.